content/gsoc/2022.md

```diff
@@ -0,0 +1,618 @@
+---
+title: "GSoC 2022"
+layout: "single"
+ShowToc: true
+TocOpen: false
+ShowReadingTime: false
+
+---
+
+**TL;DR** Jump to the [Ideas list](#project-ideas).
+
+# Introduction
+
+This year is the second time we participate as a fork - Rizin, effectively continuing the tradition since the year 2015 (as the radare2 project).
+
+## Mentors
+
+Members of the Rizin and Cutter core teams have volunteered to guide participants for GSoC’22. They were already guiding the participants for the GSoC and RSoC in past years. Please feel free to reach out to any of them in case you need any help in selecting a project.
+
+- **Anton Kochkov** Mattermost: xvilka -- [@akochkov](https://twitter.com/akochkov)
+- **Riccardo Schirone** Mattermost: ret2libc
+- **Florian Märkl** Mattermost/Telegram: @thestr4ng3r -- [@thestr4ng3r](https://twitter.com/thestr4ng3r)
+- **Antide Petit** IRC/Telegram: xarkes -- [@xarkes\_](https://twitter.com/xarkes_)
```

Comment:
Due to my current involvement in the project, I don't think I should mentor this year, so I think you can remove this line :)